<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KBJobView</class>
 <widget class="QWidget" name="KBJobView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jobs</string>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <widget class="QGroupBox" name="jobBox">
     <property name="title">
      <string>Jobs</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="KPushButton" name="dequeueButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Dequeue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="executeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>241</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
